Week1/github.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+{
+    const URL_Rep = 'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
+    function renderRepository(container, repository) {
+        let someDate = new Date(repository.updated_at);
+        container.innerHTML = "";
+        const innerTable = createAndAppend("table", container);
+        const tBody = createAndAppend("tbody", innerTable);
+        const tr = createAndAppend('tr', tBody);
+        createAndAppend('td', tr, {text: "Repository:", class: "nameRow"});
+        const tdForLink = createAndAppend('td', tr,);
```

Comment:
it's not a valid syntax to have an empty field after the last comma of your function call, nothing will happen but it may have some confusion, so if I came after a year or so, for the first look here I will say something was here and removed.

and the name you can Just call it `repositoyLink` in case you want to refer that it's a `<td>` tag you can just call it `td_repositoyLink` or as always you may have a better one :100: